plugins/dialog/Cargo.toml

```
@@ -14,6 +14,13 @@ rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 targets = ["x86_64-unknown-linux-gnu", "x86_64-linux-android"]
 
+[package.metadata.platforms.support]
+windows = { level = "full", notes = "" }
+linux = { level = "full", notes = "" }
+macos = { level = "full", notes = "" }
+android = { level = "full", notes = "" }
+ios = { level = "full", notes = "" }
```

Depends on https://github.com/tauri-apps/plugins-workspace/issues/933 being implemented otherwise we need to start with:

```suggestion
android = { level = "partial", notes = "Does not support folder picker" }
ios = { level = "partial", notes = "Does not support folder picker" }
```